test/test_op_aliases.py

```diff
@@ -150,6 +150,18 @@ def __init__(self,
     AliasInfo('true_divide_', torch.Tensor.true_divide_, 'div_', torch.Tensor.div_,
               lambda d: torch.randn(20, device=d), get_args=lambda d: (torch.rand(20, device=d) + .1,),
               decorators=(onlyCPU,)),
+    AliasInfo('swapdims', torch.Tensor.swapdims, 'transpose', torch.Tensor.transpose,
+              lambda d: torch.randn(20, 3, 2, 1, device=d), get_args=lambda d: (3, 1),
+              decorators=(onlyCPU,)),
```

Comment:
What happens when this decorator is removed? Currently we run some of these tests only on CPU to avoid a memory leak issue with CUDA tensors, but I think that only applies if an argument (from get_args) is a tensor, which shouldn't be the case here.